bcipy/simulator/data/processor_registry.py

```diff
@@ -0,0 +1,75 @@
+"""Registry for data processors."""
+import logging as logger
+from typing import List, Optional, Type
+
+from bcipy.acquisition.multimodal import ContentType
+from bcipy.signal.model.base_model import SignalModel
+# flake8: noqa
+from bcipy.simulator.data.data_process import (EegRawDataProcessor,
+                                               RawDataProcessor)
+from bcipy.simulator.demo.button_press_data_processor import \
+    ButtonPressDataProcessor
+from bcipy.task.data import EvidenceType
+
+log = logger.getLogger(__name__)
+
+PROCESSOR_TYPES = [EegRawDataProcessor, ButtonPressDataProcessor]
```

Comment:
I prefer the PEP8 recommendation of `EEGDataProcessor` when handling acronyms. I'm not sure the raw is needed when examining the emerging pattern, either.